docs/api-documentation.md

```diff
@@ -0,0 +1,130 @@
+# Claim Token API
+
+**Base URL:** `https://forest-explorer.chainsafe.dev`  
+**Endpoint:** `/api/claim_token`  
+**HTTP Method:** `GET`
+
+## Description
+
+The Claim Token API provides a simple way to request calibnet tokens from the
+faucet. This is primarily intended for developers and testers who need tokens to
+interact with the network. Users provide a valid wallet address and specify the
+token type (`faucet_info`) they wish to receive. On success, the API returns a
+transaction ID confirming the token transfer.
+
+**Key points:**
+
+- Each address may be subject to rate limiting to prevent abuse.
+- This API only distributes Calibnet `tFIL` and `tUSDFC` tokens.
+
+---
+
+## Query Parameters
+
+| Parameter     | Type   | Required | Description                                                               |
+| ------------- | ------ | -------- | ------------------------------------------------------------------------- |
+| `faucet_info` | string | Yes      | The type of token to claim. Valid values: `CalibnetFIL`, `CalibnetUSDFC`. |
+| `address`     | string | Yes      | The wallet address to receive the token.                                  |
+
+---
+
+## Status Codes
+
+| Status Code | Description                                                  |
+| ----------- | ------------------------------------------------------------ |
+| 200         | Token successfully claimed; response contains transaction ID |
+| 400         | Bad request - invalid address                                |
+| 429         | Too many requests - rate limited                             |
+| 500         | Server error; response contains error message                |
+| 418         | I'm a teapot - mainnet not supported                         |
+
```

Comment:
Add some details about the rate limiting, basically how the `claim_token` API is rate limited or we have seperate rate-limiting doc then link that. Something like:

```
| Token Type    | Cooldown Period | Daily Wallet Limit | Daily Global Limit |
|---------------|----------------|-------------------|-------------------|
| CalibnetFIL   | 60 seconds     | 2 FIL             | 200 FIL           |
| CalibnetUSDFC | 60 seconds     | 200 USDFC         | 20,000 USDFC      |
```